File: F/1295-find-numbers-with-even-number-of-digits/main.go
package main

func findNumbers(nums []int) int {
	c := 0
	for i := 0; i < len(nums); i++ {
		b := nums[i]

		if (b == 100000) || (b < 10000 && b >= 1000) || (b < 100 && b >= 10) {
			c++
		}
	}
	return c
}

func main() {
	fmt.Println("a")
}

File: F/1295-find-numbers-with-even-number-of-digits/main.rs
impl Solution {
  pub fn find_numbers(nums: Vec<i32>) -> i32 {
    nums.clone().iter_mut().fold(0, |sum, n| {
      while *n >= 100 {
        *n /= 100;
      }
      sum + if *n > 9 { 1 } else { 0 }
    })
  }
}

struct Solution {}
